### ISSUE TITLE
Include the Upload component.

### ISSUE THREAD

**Issue by fabmont** [jannikbuschke/formik-antd#120]
The upload component is quite important for some forms and maybe including this into de component list should be nice.

**Comment by fabmont** [comment]
Yeah, I'm talking about staging files. The user will select some files and when he hits the submit button, our app should send the binaries to our backend. I think it is possible to implement.

**Comment by jannikbuschke** [comment]
In the ant design docs https://ant.design/components/upload/ there is an **Upload manually** example, with which it should be rather straight forward to implement this functionality.

Im not sure if the file contents should be put into the formstate itself, or rather just into a component state.

**Comment by fabmont** [comment]
The upload component provides by Ant has the prop `onChange` which gives to you an object with the currrent file that has been uploaded to the component and the file list which is the amount of files previously uploaded. By the way, it also has a `fileList` prop that you can "store" the files, so the files that will stay into formik's values should be something like this `fileList` prop.

**Comment by fabmont** [comment]
It's a good point... I'll try this later and see how it works and I'll also try using Formik too to use the differences.

**Comment by Frank5000** [comment]
@fabmont were you able to get it working?

**Comment by jannikbuschke** [comment]
@Frank5000 I implemented an UploadFile component in my (undocumented, not yet releasable) glow library: https://github.com/jannikbuschke/glow/blob/master/core/web/src/files/upload-files.tsx#L31
There I use ant designs upload component to upload files to my backend. On successfull uploads I reference these already uploaded files in my local formik form. So when I send the form, I only reference already uploaded files, if this makes sense.

**Comment by Frostbourn** [comment]
Hi, any news on that feature?
